applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -50,6 +50,8 @@ struct le_audio_headset {
 	struct bt_conn *headset_conn;
 	struct net_buf_pool *iso_tx_pool;
 	atomic_t iso_tx_pool_alloc;
+	struct bt_codec_qos_pref pref_qos;
+	uint32_t pd;
```

Comment:
`pd `is very short. Recommend using a more descriptive name.

Reply:
Recommend: `pres_dly_us`

Reply:
Will expand, but this is the name used within the QoS structure which this variable sets.